### DISCUSSION TITLE
What is duration in the output XML?

### DISCUSSION THREAD

**Discussion by arzaatri** [Audiveris/audiveris#756] [Q&A] [answered]
Hello all, I was looking at the XML output by Audiveris and noticed a tag called `<duration>`. I would have thought this would be something like quarter/eighth/etc, but that's under the `<type>` tag. I can see that `<duration>` is similar, e.g. a quarter note's duration is twice that of an either note, but what quantity does the field represent?

**Comment by hbitteur** [comment, marked as answer]
Audiveris exports the recognized music using the de facto standard XML format described in MusicXML 4.0.
Therefore your question relates more to MusicXML than to Audiveris it self.

Reading MusicXML documentation, you will get a precise answer to your question in the definition of the [duration tag](https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/duration/).
The **duration** is a positive number of **divisions**. See both divisions tag and duration tag in [this example](https://www.w3.org/2021/06/musicxml40/musicxml-reference/examples/divisions-and-duration-elements/).
